aristamd/src/app/components/accordion/accordion.component.js

```diff
@@ -0,0 +1,54 @@
+
+var accordion = {
```

Comment:
this is not written on ES6 because we are looking for a simpler pass down to web-app